### ISSUE TITLE
More permanent storage for game states

### ISSUE THREAD

**Issue by cpjolicoeur** [mrroach/CentralServer#11]
To make game saving/reloading easier and more reliable and to keep the memory footprint of the process down, we should probably store and load the game states from redis or something similar.  Games that have been completed can probably be archived in something like a sqlite (or any kind) of database, but moving game saves outside of the local processes memory will allow for better persistence and memory usage.

If agreed, I can probably tackle this

**Comment by mrroach** [comment]
Yup, that sounds good. My lazy thought has been to just pickle the game
objects, but extra credit for a real serialization format ;-)

**Comment by cpjolicoeur** [comment]
Yeah, I started on it this morning and was just going to cPickle them directly.  Might just try to get that going before writing some kind of nicer/slicker custom serializer for now.

**Comment by mrroach** [comment]
SGTM
